<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set_Dialog</class>
 <widget class="QDialog" name="set_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>481</width>
     <height>351</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>显示设置</string>
    </attribute>
    <widget class="QLabel" name="pifu_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>33</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>窗体颜色</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>单词字体</string>
     </property>
    </widget>
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>203</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>窗口大小</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="w_with">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>55</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="w_len">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>55</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>单词颜色</string>
     </property>
    </widget>
    <widget class="QPushButton" name="word_color">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/choosecolor.png</normaloff>image/choosecolor.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>窗口透明度</string>
     </property>
    </widget>
    <widget class="QSlider" name="tm_horizontalSlider">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>280</y>
       <width>231</width>
       <height>19</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="tm_label">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>280</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="windowColor_pushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/choosecolor.png</normaloff>image/choosecolor.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>词典设置</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>单词排序</string>
     </property>
    </widget>
    <widget class="QComboBox" name="wordsort_comboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>随机</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按单词首字母排序 </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>生词保存目录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="WordSavePath_lineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browse_bt">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>网络设置</string>
    </attribute>
    <widget class="QCheckBox" name="WebisOpen_checkBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>打开网络查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_bt">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>添加&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Name_lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="handAdd_bt">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>280</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>手动添加</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>URL</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="URL_lineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>240</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="webSet_listWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>81</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="webURL_listWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="moveWeb_pushButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;移除</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>网络词库</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel_bt">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_bt">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>370</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cancel_bt</tabstop>
  <tabstop>save_bt</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>w_with</tabstop>
  <tabstop>w_len</tabstop>
  <tabstop>word_color</tabstop>
  <tabstop>tm_horizontalSlider</tabstop>
  <tabstop>windowColor_pushButton</tabstop>
  <tabstop>wordsort_comboBox</tabstop>
  <tabstop>WordSavePath_lineEdit</tabstop>
  <tabstop>browse_bt</tabstop>
  <tabstop>WebisOpen_checkBox</tabstop>
  <tabstop>add_bt</tabstop>
  <tabstop>Name_lineEdit</tabstop>
  <tabstop>handAdd_bt</tabstop>
  <tabstop>URL_lineEdit</tabstop>
  <tabstop>webSet_listWidget</tabstop>
  <tabstop>webURL_listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
